### PR TITLE
Fix/replace icon

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Replace placeholder icons with new style kit icons.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeError.swift [hunk 1]
```diff
@@ -43,19 +45,20 @@ enum PasscodeError: CaseIterable {
...
+        
+        let attributedString: NSAttributedString = NSAttributedString(string: message) && font
+        
+        return NSAttributedString(attachment:textAttachment) + attributedString
 
-        return attributedString
+    }
+    
+    var descriptionWithInvalidIcon: NSAttributedString {
+        return description(icon: .circleCross, color: UIColor(rgb: (150, 153, 156)), font: .smallRegularFont)
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeError.swift:58]: Please name this color.

File: Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeError.swift [hunk 2]
```diff
@@ -43,19 +45,20 @@ enum PasscodeError: CaseIterable {
...
     }
 
-    //TODO paint code icon
     var descriptionWithPassedIcon: NSAttributedString {
-
-        let attributedString: NSAttributedString = NSAttributedString(string: "✅" + message)
-
-        return attributedString
+        return description(icon: .circleTick, color: UIColor(rgb: (2, 193, 1)), font: .smallSemiboldFont)
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Overlay/CustomAppLock/Setup/PasscodeError.swift:62]: Please name this color.

File: Wire-iOS/Sources/UserInterface/Components/CGFloat+Const.swift [hunk 3]
```diff
@@ -26,6 +26,13 @@ extension Float {
     }
 }
 
+extension StyleKitIcon.Size {
+    enum CreatePasscode {
+        static let iconSize: StyleKitIcon.Size = 11
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/CGFloat+Const.swift:31]: Why are these custom sizes needed? Are we compensating for something?

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Components/CGFloat+Const.swift:31]: I took a look at the Paintcode file and the new eyeSlash icon looks squished vertically. Could we maybe fix this my transforming it to match the existing eye icon?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/CGFloat+Const.swift:31]: This is for the icon size only use in the text attachment. I changed to `.custom(11)` for readability

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/CGFloat+Const.swift:31]: I will add one more new eye icon for this screen

**Pull request merged**